patches/asmpatchhandler.py

```diff
@@ -39,7 +40,7 @@
 # Adds a patch to nnSdk to route all vfprintf calls to the debug output
 # These will be printed to the console on yuzu
 # These prints will spam the console so don't leave this set to True
-ASM_DEBUG_PRINT = False
+ASM_DEBUG_PRINT = True
```

Comment:
Is this meant to always be on now or was this just leftover from testing?

Reply:
Yeah, this is meant to be left on. This PR is based on the debug print one (doing this without it would have been a nightmare xD).

Only our custom prints show up now so the console doesn't get spammed with useless junk any more. Leaving it on is more for dev convenience. Didn't want to get rid of the option to turn off the debug prints just in case but there's not really a reason to turn them off anymore.

Reply:
The comment above the variable should probably be updated tho